docs/docs/api/get.md

```diff
@@ -17,6 +17,8 @@ after the `view` or `producer` was triggered. It works the same way as
 2. A value is needed at a later time since producer was triggered, e.g while
    performing an asynchronous operation
 
+## Example
+
 For example, if the state looks like:
```

Comment:
Also, Get can receive an argument for adding to the path at runtime:
```
getValue = Get.foo.bar[Param.keyName].value
---
getValue({ keyName: "baz" })
```

Reply:
Also, values received from the state are cloned and are not delivered through reference UNLESS they are non-serializable values. For example, for a Blob or a Stream data type, the reference will be kept and the value will not be cloned.

Reply:
> Also, Get can receive an argument for adding to the path at runtime:

Isn't this the case with `Update` as well? Do all Engine operators allow setting `Param`s from their second argument?

> Also, values received from the state are cloned and are not delivered through reference UNLESS they are non-serializable values. For example, for a Blob or a Stream data type, the reference will be kept and the value will not be cloned.

Assuming this is the behavior for both Get and Observe, I think it'd be better to document this in `producer` page instead.

Reply:
@dumconstantin Do `Param`s act as triggers? Do a change to `Param` need to be guarded for infinite loops? Or do they provide the new value on next execution of a producer/reducer?

Reply:
> Assuming this is the behavior for both Get and Observe, I think it'd be better to document this in producer page instead.

I had a change of heart and chose to document it in both Get and Observe instead. I think this is the really the behavior of the accessor operator. And we might want to add more accessor operators with different behavior in future (e.g MutableGet).

Reply:
Regarding the cloning, yes having it on Get and Observe i also think its the best option.

Regarding Param, both Get and Update can use it so i think we should document this feature on both pages.

Param woudn't be able to trigger infinite loops because you can't use Observe with it. They are used at the invocation point so they do not exist past the get() or update() call.